### PR TITLE
Updating the 'Job Hunting' section and fixing a few spelling errors

### PR DESCRIPTION
**Pull request by scouttyg**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -372,7 +372,7 @@
                 </li>
                 <li>
                     <a href="http://fordschool.umich.edu/events/upcoming">Gerald R. Ford School of Public Policy Science, Technology, and Public Policy Program Events</a>
-                    <p>The University of Michigan's Gerald R. Ford School of Public Policy's science and technology events, includes public events. (You'll need to filter their calendar for the approrpiate topics and public access.)</p>
```
Comment (scouttyg) [inline index.html:375]: Just a small spelling error here -- approrpiate -> appropriate

File: index.html [hunk 2]
```diff
@@ -528,8 +528,12 @@
                 <p>Talent consultants and recruiters/matchmakers focused on supporting Michigan technology companies with their talent needs.</p>
               </li>
               <li>
-                <a href="https://angel.co/michigan">AngleList - Ann Arbor Startups</a>
```
Comment (scouttyg) [inline index.html:531]: Another small spelling correction - AngleList -> AngelList

File: index.html [hunk 3]
```diff
@@ -528,8 +528,12 @@
...
-                <p>A listing of Ann Arbor startups, including job openings.</p>
+                <a href="https://angel.co/michigan">AngelList - Michigan Startups</a>
```
Comment (scouttyg) [inline index.html]: The link for AngelList had the text "Ann Arbor Startups", but linked to https://angel.co/michigan, which is all Michigan startups. I updated the text here to instead say 'Michigan Startups', but we could also link to https://angel.co/ann-arbor instead and keep the old text.

File: index.html [hunk 4]
```diff
@@ -528,8 +528,12 @@
...
+                <p>A listing of Michigan startups, including job openings.</p>
+              </li>
+              <li>
+                <a href="https://www.truejob.com/locations/ann-arbor/jobs">TrueJob - Ann Arbor Startups</a>
```
Comment (scouttyg) [inline index.html:535]: @mwkling and I created [TrueJob](http://www.truejob.com/) here in Ann Arbor, and we have the biggest list of Ann Arbor startup jobs of any other site.

File: index.html (hunk 3 above)
Comment (jshwlkr) [inline index.html]: Yeah, could you just switch the link and resubmit the merge request, please?

File: index.html [hunk 5]
```diff
@@ -528,9 +528,13 @@
                 <p>Talent consultants and recruiters/matchmakers focused on supporting Michigan technology companies with their talent needs.</p>
               </li>
               <li>
-                <a href="https://angel.co/michigan">AngleList - Ann Arbor Startups</a>
+                <a href="https://angel.co/ann-arbor">AngleList - Ann Arbor Startups</a>
```
Comment (scouttyg) [inline index.html:531]: Fixed this commit so instead we don't change the text, we instead change the link!

File: index.html [hunk 6]
```diff
@@ -528,9 +528,13 @@
...
                 <p>A listing of Ann Arbor startups, including job openings.</p>
               </li>
+              <li>
+                <a href="https://www.truejob.com/locations/ann-arbor/jobs">TrueJob - Ann Arbor Startups</a>
```
Comment (scouttyg) [inline index.html:535]: @mwkling and I created TrueJob here in Ann Arbor, and we have the biggest list of Ann Arbor startup jobs of any other site.

**Comment by scouttyg** [comment]
Updated the pull request! Our old comments were deleted/hidden so I added a few back, but I made it so instead of changing the text for AngelList to Michigan we instead link to the correct AngelList link for Ann Arbor

**Pull request merged**